component/distributed_cache/utils_test.go

```diff
@@ -0,0 +1,80 @@
+/*
+	_____           _____   _____   ____          ______  _____  ------
```

Comment:
These lines are misaligned in most files, later if we go to fix the copy right version it might create issues.